environment/git/pre-commit

```diff
@@ -41,11 +41,5 @@ for hook in $HOOKS; do
     if [ $? != 0 ]; then
       exit 1
     fi
-  else
-    echo "Error: file ${hook} not found."
```

Comment:
Could this be turned into a warning?

Reply:
Yes, but I don't think it would be very useful for most cases -- maybe for people who are adding new hooks.

Reply:
That's a good point.  I would lean towards removing it.